README.md

```diff
@@ -95,5 +95,8 @@ Welcome to the tribe.
         - [How to Deal with Organizational Chaos](3-Advanced/Serving-Your-Team/11-How to Deal with Organizational Chaos.md)
 4. [Glossary](4-Glossary.md)
 5. [Appendix A - Bibliography/Websiteography](5-Bibliography.md)
-6. [Appendix B - History (As of February 2003)](6-History.md)
-7. [Appendix C - GNU Free Documentation License](LICENSE)
+6. [Appendix B - History (As of January 2016)](6-History.md)
+6. [Appendix C - Contributions (As of January 2016)](7-Contributions.md)
+
```

Comment:
Do you think we should continue to include a link to the license - Appendix D perhaps?